tools/contractdeployer/main/main.go

```diff
@@ -8,7 +8,12 @@ import (
 func main() {
 	log.SetLogLevel(log.DisabledLevel)
 	config := contractdeployer.ParseConfig()
-	deployer := contractdeployer.NewContractDeployer(config)
+	deployer, err := contractdeployer.NewContractDeployer(config)
+	if err != nil {
+		// todo: why is this log level stuff setup in this way (why not print here or use logs everywhere)
```

Comment:
Good question.

Logs are disabled so that the only output is the contract deployer address.
If there are logs, then it complicated the retrieval of the contract address from the output logs.

Might be worth improving, for sure.